index.html

```diff
@@ -1,20 +1,320 @@
+              src="https://source.unsplash.com/random"
+              class="col-sm-4 col-md-4 col-lg-5"
+              alt="card-img"
+            />
+            <div
+              class="teams-card-txt d-flex flex-column gap-3 col-sm-6 col-md-6 col-lg-5"
+            >
+              <p class="fw-bold fs-4 text-dark">John Doe</p>
+              <p class="fst-italic">Director, Chairman hevons Farms</p>
+              <p class="fs-small mt-1 text-dark">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit
+                non sapiente in.
+              </p>
+            </div>
+          </div>
+          <div
+            class="teams-card-col d-flex gap-4 col-sm-12 col-md-12 col-lg-6 pb-1"
+          >
+            <img
+              src="https://source.unsplash.com/random"
+              class="col-sm-4 col-md-4 col-lg-5"
+              alt="card-img"
+            />
+            <div
+              class="teams-card-txt d-flex flex-column gap-3 col-sm-6 col-md-6 col-lg-5"
+            >
+              <p class="fw-bold fs-4 text-dark">John Doe</p>
+              <p class="fst-italic">Director, Chairman hevons Farms</p>
+              <p class="fs-small mt-1 text-dark">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit
+                non sapiente in.
+              </p>
+            </div>
+          </div>
+          <div
+            class="teams-card-col d-flex gap-4 col-sm-12 col-md-12 col-lg-6 pb-1"
+          >
+            <img
+              src="https://source.unsplash.com/random"
+              class="col-sm-4 col-md-4 col-lg-5"
+              alt="card-img"
+            />
+            <div
+              class="teams-card-txt d-flex flex-column gap-3 col-sm-6 col-md-6 col-lg-5"
+            >
+              <p class="fw-bold fs-4 text-dark">John Doe</p>
+              <p class="fst-italic">Director, Chairman hevons Farms</p>
+              <p class="fs-small mt-1 text-dark">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit
+                non sapiente in.
+              </p>
+            </div>
+          </div>
+        </div>
+        <button
+          type="button"
+          class="btn btn-outline-warning w-75 mx-auto text-warning p-4 d-lg-none d-flex align-items-center justify-content-center gap-4"
+        >
+          More
+          <span class="material-icons icons"> keyboard_arrow_down </span>
+        </button>
+      </section>
+      <!-- Partners Section -->
+      <section
+        class="bg-dark d-lg-flex flex-column gap-4 col-12 py-3 d-none partners"
+      >
+        <h3 class="h3 text-center">Our Partners</h3>
+        <div class="line"></div>
+        <div class="d-flex justify-content-evenly col-12">
+          <img src="./img/bank.png" alt="bankIcon" />
+          <img src="./img/insurance.png" alt="insuranceIcon" />
+          <img src="./img/google.webp" alt="googleIcon" />
+        </div>
+      </section>
```

Comment:
- [ ] It would also be good to add more padding to this section too as well as use the correct color for its heading 😄 
- Your design:
![Capture5](https://user-images.githubusercontent.com/85113567/181269860-6f5d629a-3040-4641-8dc3-f57120e55465.PNG)
- Required design:
![Capture7](https://user-images.githubusercontent.com/85113567/181269914-9cdf5a65-39ff-4b96-9b8f-7d99cd934e35.PNG)

Reply:
Resolved